rest/src/main/scala/io/udash/rest/annotations.scala

```diff
@@ -240,6 +241,38 @@ class Cookie(@defaultsToName override val name: String = RestParamTag.paramName)
 class Body(@defaultsToName override val name: String = RestParamTag.paramName)
   extends rpcName(name) with RestParamTag
 
+/**
+  * Like [[Query]] but indicates that the parameter is optional.
```

Comment:
Let's add some information about what that indication actually does / changes (to all these), so it's more obvious why someone would want to use this.